images/win/scripts/Installers/Install-JavaTools.ps1

```diff
@@ -85,14 +81,8 @@ setx MAVEN_OPTS $maven_opts /M
 $uri = 'https://ayera.dl.sourceforge.net/project/cobertura/cobertura/2.1.1/cobertura-2.1.1-bin.zip'
 $coberturaPath = "C:\cobertura-2.1.1"
 
-cd $env:TEMP
-
-Invoke-WebRequest -UseBasicParsing -Uri $uri -OutFile cobertura.zip
-
-# Expand the zip
-Expand-Archive -Path cobertura.zip -DestinationPath "C:\" -Force
-
-# Deleting zip folder
-Remove-Item -Recurse -Force cobertura.zip
+Start-DownloadWithRetry -Url $uri -Name "cobertura.zip" |
```

Comment:
Why do we use so complex syntax?
$cPath = Start-DownloadWithRetry -Url $uri -Name "cobertura.zip"
Expand-Archive -Path $cPath -DestinationPath "C:\"

Reply:
Fixed